_includes/why.html

```diff
@@ -3,20 +3,14 @@
 
   <h2 id="why" class="section-header">Why</h2>
 
-  <p>
-    We would like to help improve diversity and the gender ratio in the Cape Town tech community.
-  </p>
+  <p>We want to push for all kinds of diversity in the Cape Town tech community: gender, race, sexual orientation, ability, and class. Women were the first population that RailsBridge focused on, but aren't the last.</p>
```

Comment:
re race: :+1:, changed.

re class: I guess what we mean is socioeconomic status. Happy to ditch it from the blurb if that's better, though.